tests/layer_tests/pytorch_tests/test_quantized_linear.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+class TestQuantizedLinear(PytorchLayerTest):
+    def _prepare_input(self, input_shape=(2, 2)):
+        return (np.random.randn(*input_shape).astype(np.float32),)
+
+    def create_model(self, weight_shape, is_bias):
+
+        class aten_quantized_linear(torch.nn.Module):
+            def __init__(self, weight_shape, is_bias):
+                super(aten_quantized_linear, self).__init__()
+                if is_bias:
+                    self.linear = torch.ao.nn.quantized.Linear(weight_shape[-1], weight_shape[0], True)
```

Comment:
Please test different values of `scale` and `zero_point` for linear op.

Reply:
Added tests for different scale/zero-point combinations for linear operator. As for testing different weights/dtypes it is still in progress, I couldn't simply overwrite default weight tensor quantization schema (by default, quantized per tensor and qint8 dtype) with different one, like done with bias, probably it would require to use different initializer for linear than used in current tests